vscode/extension/README.md

```diff
@@ -9,7 +9,7 @@ WSO2 Ballerina Integrator VS Code extension is a tool that is used alongside the
 
 ## Quick Start
 - **Step 1:** Install the WSO2 Ballerina Integrator extension for Visual Studio Code.
-- **Step 2:** Use the command `Ballerina Integrator: Dashboard` through the command palette to start creating projects and modules using multiple different templates provided.
+- **Step 2:** Use the command `Ballerina Integrator: Dashboard` through the command palette ([Command] + [Shift] + [P] on macOS and [Ctrl] + [Shift] + [P] on Windows/Linux) to start creating projects and modules using multiple different templates provided.
```

Comment:
This command is there below again under "Useful Commands", we can remove that if it looks like getting duplicated.

Reply:
Since this is required for the other commands to be run as well, shall we leave it as is?